<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zoomControlWidget</class>
 <widget class="QWidget" name="zoomControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>85</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="coordLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>123</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="positionDial">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>86</width>
     <height>86</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="notchTarget">
    <double>25.699999999999999</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="bSlider" name="speedSlider">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>95</y>
     <width>76</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>bSlider</class>
   <extends>QSlider</extends>
   <header location="global">bslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
